frontend/src/pages/ApplicationsPage.tsx

```diff
@@ -92,7 +92,7 @@ const ApplicationsPage: React.FC<ApplicationsPageProps> = ({
         <PageSection isFilled>
           <EmptyState variant={EmptyStateVariant.large} data-id="loading-empty-state">
             <Spinner size="xl" />
-            <Title headingLevel="h4" size="lg">
+            <Title headingLevel="h1" size="lg">
```

Comment:
Is this going to break a11y?

Reply:
this fixes a11y. we got an issue because we are stepping down to h4 before h1. storybook is testing a11y and this fixed it